### ISSUE TITLE
Grid border color

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/calendar#4767]
### Steps to reproduce

1. Run latest server master
2. Run latest Calendar main

### Expected behavior

The usual

### Actual behaviour

![Bildschirmfoto vom 2022-11-23 17-02-04](https://user-images.githubusercontent.com/1374172/203593172-fd5c5a7f-39b6-46c9-b753-f40956587024.png)

### Additional info

This change was done with https://github.com/nextcloud/server/pull/35315. @Pytal do you know if the border contrast ratio should apply for the calendar grid as well? If not, we could have lower contrast value for the calendar grid. If the contrast should be the way it is now we can close the ticket.

**Comment by Pytal** [comment]
Not sure, should different border colors be used for the calendar grid @nimishavijay?

**Comment by nimishavijay** [comment]
I would say the old more subtle border color works, since in most cases there are events for a day and even if not it is indicated by the date number anyway. cc @jancborchardt on this when he gets back

**Comment by Pytal** [comment]
- [ ] Clarify with accessibility reviewer

**Comment by obrb** [comment]
I honestly doubt that the Boarder Contrast Ratio, mentioned in #35 also refers to calendar grids and line separators, at least the [document](https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/Contrast_(Minimum)) linked in [#35315](https://github.com/nextcloud/server/pull/35315) does not explicitly mention them. Either way I think it's ugly ;-) and unnecessary for content separators, especially since there is separate High Contrast Theme in Nextcloud, which already does the same thing. See also my post in the forums: https://help.nextcloud.com/t/rcs-of-25-0-2-24-0-8-23-0-12/150660/2?u=bb77
